content/docs/rfcs/71/README.md

```diff
@@ -0,0 +1,436 @@
+---
+slug: 71
+title: 71/STATUS-PUSH-NOTIFICATION-SERVER
+name: Push Notification Server
+status: raw
+category: Standards Track
+editor: Jimmy Debe <jimmy@status.im>
+contributors: Andrea Maria Piana <andreap@status.im>
+---
+
+
+# Abstract
```

Comment:
Please use [sembr](https://sembr.org) for text so it is easier to edit without conflicts later